jbcli/jbcli/tests/test_cli.py

```diff
@@ -1012,57 +1012,6 @@ def test_stop(self, dockerutil_mock):
             call.halt()
         ]
 
-    @patch('jbcli.cli.jb.dockerutil')
-    def test_start(self, dockerutil_mock):
-        dockerutil_mock.is_running.return_value = False
-        dockerutil_mock.ensure_home.return_value = True
-        runner = CliRunner()
-        result = runner.invoke(cli, ['start', '--noupgrade'])
-        assert result.exit_code == 0
-        assert dockerutil_mock.mock_calls == [
-            call.is_running(),
-            call.pull(tag=None),
-            call.up(env=ANY)
-        ]
-
-    @patch('jbcli.cli.jb.dockerutil')
-    def test_start_noupgrade(self, dockerutil_mock):
-        dockerutil_mock.is_running.return_value = False
-        dockerutil_mock.ensure_home.return_value = True
-        runner = CliRunner()
-        result = runner.invoke(cli, ['start', '--noupgrade'])
-        assert result.exit_code == 0
-        assert dockerutil_mock.mock_calls == [
-            call.is_running(),
-            call.pull(tag=None),
-            call.up(env=ANY)
-        ]
-
-    @patch('jbcli.cli.jb.dockerutil')
-    def test_start_noupdate(self, dockerutil_mock):
-        dockerutil_mock.is_running.return_value = False
-        dockerutil_mock.ensure_home.return_value = True
-        runner = CliRunner()
-        result = runner.invoke(cli, ['start', '--noupdate', '--noupgrade'])
-        assert dockerutil_mock.mock_calls == [
-            call.is_running(),
-            call.up(env=ANY)
-        ]
-        assert result.exit_code == 0
-
-    @patch('jbcli.cli.jb.dockerutil')
-    def test_start_already_running(self, dockerutil_mock):
-        dockerutil_mock.is_running.return_value = True
-        dockerutil_mock.ensure_home.return_value = True
-        runner = CliRunner()
-        result = runner.invoke(cli, ['start', '--noupgrade'])
-
-        assert 'An instance of Juicebox is already running' in result.output
-        assert result.exit_code == 0
-        assert dockerutil_mock.mock_calls == [
-            call.is_running()
-        ]
-
```

Comment:
little confused why these are being removed, they should still be valid right?

Reply:
so, I *thought* there were separate freshstart tests, but now I realize they've never existed. So I will update these tests to test freshstart (now start).